src/main/java/com/emc/object/s3/request/InitiateMultipartUploadRequest.java

```diff
@@ -33,13 +33,15 @@
 import com.emc.object.s3.bean.CannedAcl;
 import com.emc.object.util.RestUtil;
 
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 public class InitiateMultipartUploadRequest extends S3ObjectRequest {
     private S3ObjectMetadata objectMetadata;
     private AccessControlList acl;
     private CannedAcl cannedAcl;
+    private HashMap<String, String> tagging;
```

Comment:
Should leverage `ObjectTagging` that's already defined, and update getter/setter accordingly.